tests/test_voice_handler.py

```
@@ -178,21 +154,42 @@ async def test_skip_current_message(self, voice_handler: VoiceHandler) -> None:
+        except asyncio.TimeoutError:
             pytest.fail("Test timed out - skip_current_message operation took too long")
 
     @pytest.mark.asyncio
     async def test_clear_all_queues(self, voice_handler: VoiceHandler) -> None:
         """Test clearing all queues."""
         # Add items to both queues
         await voice_handler.synthesis_queue.put({"text": "syn1"})
-        await voice_handler.audio_queue.put(("path1", "group1", 1, 1024))
+        await voice_handler.audio_queue.put(AudioItem("path1", "group1", 1, 0, 1024))
 
         cleared: int = await voice_handler.clear_all()
```

Ruff (0.12.2)

Local variable `cleared` is assigned to but never used

(F841)